CHANGELOG.md

```diff
@@ -22,6 +23,100 @@
  - [0.0.5](#005---20180602)
  - [0.0.4 and prior](#004-and-prior)
 
+## [0.10.0] - someday
+
+#### Breaking changes
+
+- Ingress resources now require `kubernetes.io/ingress.class` annotations by
+  default. Kong recommends adding this annotation to Ingresses that previously
+  did not have it, but you can override this change and instruct the controller
+  to process Ingresses without this annotation if desired. See the [ingress
+  class documentation](https://example.com/link-tbd) for details.
+  [#767](https://github.com/Kong/kubernetes-ingress-controller/pull/767)
+- KongConsumer resources now require `kubernetes.io/ingress.class` annotations
+  by default. This change can also be overriden using a flag.
+  [#767](https://github.com/Kong/kubernetes-ingress-controller/pull/767)
+- TCPIngress resources now require `kubernetes.io/ingress.class` annotations.
+  This change _cannot_ be overriden.
+  [#767](https://github.com/Kong/kubernetes-ingress-controller/pull/767)
+- CA certificate secrets now require `kubernetes.io/ingress.class` annotations.
+  This change _cannot_ be overriden.
+  [#815](https://github.com/Kong/kubernetes-ingress-controller/pull/815)
+- Removed support for global KongPlugin resources. You must now use
+  KongClusterPlugin resources for global plugins. You should run
+  `kubectl get kongplugin -l global=true --all-namespaces` to list existing
+  global KongPlugins to find and convert them before upgrading. The controller
+  will also log a warning if it finds any global KongPlugins that are still in
+  place.
+  [#751](https://github.com/Kong/kubernetes-ingress-controller/pull/751)
+
+
+#### Added
+
+- Added support for [Ingress
+  v1](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1453-ingress-api#summary-of-the-proposed-changes).
+  [#TBD](https://github.com/Kong/kubernetes-ingress-controller/pull/TBD).
```

Comment:
Please instead point to the umbrella issue: https://github.com/Kong/kubernetes-ingress-controller/issues/590